src/test/scala/latis/service/hapi/TimeParseSuite.scala

```diff
@@ -2,57 +2,54 @@ package latis.service.hapi
 
 import java.time.LocalDateTime
 
+import munit.CatsEffectSuite
 import org.scalactic.Equality
-import org.scalatest.Assertion
-import org.scalatest.flatspec._
+import org.scalatest.Assertions.convertToEqualizer
 
-class TimeSpec extends AnyFlatSpec {
+class TimeParseSuite extends CatsEffectSuite {
 
-  val expected = LocalDateTime.of(2018, 1, 1, 0, 0, 0)
+  val expected: LocalDateTime = LocalDateTime.of(2018, 1, 1, 0, 0, 0)
 
-  def testParse(str: String): Assertion =
-    assert {
-      expected === Time.parse(str).getOrElse {
-        fail(s"Failed to parse time string: $str")
-      }
+  def testParse(str: String): Unit =
+    Time.parse(str) match {
+      case Some(t) => assert(t === expected)
```

Comment:
Should we prefer `assertEquals`?

Reply:
From the existing comment on the `implicit val` that implements `===`:
  // The `equals` method should be used to compare `LocalDateTime`
  // instances. See the Javadocs.
So I was wary to change it.

Reply:
I suppose it depends on whether `assertEquals` is doing a `==` or a `.equals` behind the scenes...

Reply:
Seems to work fine